weasyl/controllers/moderation.py

```diff
@@ -17,6 +17,19 @@ def modcontrol_(request):
 
 
 @moderator_only
+def modcontrol_auditlog_(request):
+    form = request.web_input(username=None, staff=None, start_date=None, end_date=None)
+    
```

Comment:
There’s whitespace here; flake8 should report this sort of thing. (We ought to run it in addition to tests and coverage on pull requests, really.)